packages/account/src/test-utils/launchNode.test.ts

```diff
@@ -29,6 +29,10 @@ vi.mock('fs', async () => {
  * @group node
  */
 describe('launchNode', () => {
+  beforeEach(() => {
+    vi.spyOn(console, 'log').mockImplementation(() => {});
+  });
```

Comment:
How come your adding this in `beforeEach` if it's only needed in one test? Seems like unnecessary addition of logic for most tests.

Reply:
This was my lazy attempt at suppressing our noisy CI.

A better solution: edd959d1227c6e65f4dd4ec6520f9827a8e07a34